projects/ngx-sub-form/src/lib/shared/ngx-sub-form-utils.ts

```diff
@@ -131,13 +109,6 @@ export class MissingFormControlsError<T extends string> extends Error {
   }
 }
 
-export const NGX_SUB_FORM_HANDLE_VALUE_CHANGES_RATE_STRATEGIES = {
```

Comment:
This is still a used feature in the current version, and was never deprecated.

Reply:
Indeed, fixed (I had to update the type as it was using an old one)